crates/next-api/src/app.rs

```diff
@@ -694,6 +694,9 @@ fn server_utils_module() -> Vc<RcStr> {
     Vc::cell("server-utils".into())
 }
 
+#[turbo_tasks::value(transparent)]
+struct OutputAssetsWithAvailability((Vc<OutputAssets>, Vc<AvailabilityInfo>));
```

Comment:
I did this because I have this function parameter in `ClientReferenceManifest::build_output`
```
	rsc_app_entry_chunks_availability: Vc<AvailabilityInfo>,
```
so I needed the cell anyway.